schema.sql

```diff
@@ -8,3 +8,5 @@ CREATE TABLE animals (
   neutered boolean ,
   weight_kg decimal ,
 );
+
+ALTER TABLE animal ADD species VARCHAR(255);
```

Comment:
- Please, kindly check your table name, as you have `animals` and not `animal` because, according to your `schema.sql` file and in your database, the table `animal` does not exist. Please Update this everywhere you've used the `animal` as opposed to `animals`. Also, the table keeps track of the different animals, therefore, it should be pluralized and not singular. Please update it **EVERYWHERE YOU USED IT. _This means you have to update all your queries, data, and the screenshot in your PR_** :pray: